Backend/src/MangoTango.Api/Entities/WhitelistRequest.cs

```diff
@@ -68,12 +98,12 @@ private static async Task<string> GetJavaUuidAsync(string username)
             var data = await http.GetAsync($"https://api.mojang.com/users/profiles/minecraft/{username}");
 
             if (data.StatusCode != HttpStatusCode.OK)
-                throw new Exception("Invalid Minecraft Java Username");
-
-            var json = JsonObject.Parse(await data.Content.ReadAsStringAsync());
-            var uuid = json["id"].GetValue<string>();
+                throw new InvalidOperationException("Invalid Minecraft Java Username");
 
-            return new Guid(uuid).ToString();
+            var json = await JsonSerializer.DeserializeAsync<JsonElement>(data.Content.ReadAsStream());
+            return !json.TryGetProperty("id", out var uuid)
```

Comment:
This should actually be tested, same with GetBedrockUuidAsync. Everything looks fine but I'm still unsure about it